spec/spec_helper.rb

```diff
@@ -125,7 +121,6 @@ def sample_data(file_name)
   config.filter_run_excluding :chef_gte_12_5_only => true unless chef_gte_12_5?
```

Comment:
everything here but gte13 and the 14s can go away

Reply:
Thanks @tas50, Updated.